org.blender.Blender.json

```diff
@@ -15,6 +15,12 @@
         "--filesystem=host"
     ],
     "add-extensions": {
+        "org.blender.Blender.Codecs": {
+            "directory": "lib/codecs",
+            "add-ld-path": "lib",
+            "bundle" : true,
+            "autodelete" : true
+        },
         "org.freedesktop.Platform.ffmpeg-full": {
```

Comment:
@bochecha is `org.freedesktop.Platform.ffmpeg-full` still necessary?

Reply:
Both should work.

The ffmpeg contained in `org.blender.Blender.Codecs` has priority over the one in `org.freedesktop.Platform.ffmpeg-full` which itself has priority over the one in `org.freedesktop.Platform`.

So depending on what is installed on a user machine, then the user gets the codecs in one extension, the other, or the runtime.

At least that's the intention.

What would be the point of removing this line?

Reply:
Just curiosity. Thanks!